components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -237,6 +254,7 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 			<div class="d2l-applied-filters-wrapper">
 				<span id="d2l-applied-filters-label" class="d2l-applied-filters-applied-filters-label d2l-body-compact">${this.localize('appliedFilters')}</span>
 				${filters}
+				<d2l-button-subtle text="${this.localize('clearFilters')}" ?hidden="${this._selectedEntries.length < CLEAR_FILTERS_THRESHOLD}" @click="${this._clearFiltersClicked}"></d2l-button-subtle>
```

Comment:
It's generally worth adding this rule to the css when you use `hidden`, without it there are ways for the component to be "hidden" but still show:
```
			[hidden] {
				display: none !important;
			}
```

Also PSA, this is probably one of the only places you can use the `!important` keyword without me asking you to remove it in code review 😛

Reply:
I added the css, I assumed the right place to put it was in the `styles` function